docs/architecture/api-specs-support.md

```diff
@@ -0,0 +1,35 @@
+# API Specifications Support
+
+## Overview
+
+This document describes current and planned api specification support in Kyma components.
+
+## Currently supported api specification versions and formats
+
+Following table shows currently supported api specification versions and formats in different components.
+
+Last row describes support for extracting api specifications from archive files.
+
+| | Console UI & Compass UI (rendering) | Rafter (storing) |  Compass UI (uploading) | Compass Backend (storing) | Runtime Agent (storing)
```

Comment:
How Rafter (storing) and Runtime Agent (storing) differs?

maybe:
1. Rafter (storing) -> Runtime Rafter  (storing). In both tables.
1. will be nice to see red/gree rows in every column about format/version/validation/conversion. By that we will see which column is a most limited one. Think how to visualize that.

Reply:
Not sure if we should mention the Rafter here. We use it only as a backing service and in terms of our usecases we user nevert interacts with the Rafter directly.